soh/include/alloca.h

```diff
@@ -1,7 +1,7 @@
 #ifndef ALLOCA_H
 #define ALLOCA_H
 
-void* alloca(u32);
+// void* alloca(u32);
```

Comment:
Was this line needed for win32? Should it be ifdefed?

Reply:
It wasn't and `alloca` should not be used anyway so technically this whole file can be removed but I feel like that would belong in another PR.